.gitignore

```diff
@@ -2,3 +2,4 @@ _site/
 .sass-cache/
 .idea
 vendor
+.DS_Store
```

Comment:
What is this doing? Is it needed?

Reply:
Ok I have removed it. 
The file is OSX system file, so potentially it might be helpful for developers with OSX environment.